mu-plugins/plugin-tweaks/youtube-shortcode.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Add a shortcode for nocookie youtube embeds.
+ *
+ * These are not supported in the core embed block or shortcode.
+ * See https://core.trac.wordpress.org/ticket/44610
+ */
+
+namespace WordPressdotorg\MU_Plugins\Plugin_Tweaks\Youtube_Shortcode;
+
+add_shortcode( 'youtube-nocookie', __NAMESPACE__ . '\render' );
+
+/**
+ * Render the youtube iframe.
+ *
+ * The shortcode content is the URL, checked against a safelist of
+ * `youtube-nocookie` domains. Attributes can be `width`, `height`,
+ * and `title.
+ *
+ * @param array  $attr    Shortcode attributes array, can be empty if the original arguments string cannot be parsed.
+ * @param string $content Content inside shortcode tags.
+ * @param string $tag     Shortcode name.
```

Comment:
[Core uses the parameter name](https://developer.wordpress.org/reference/functions/shortcode_atts/#:~:text=in%20shortcode%20tag.-,%24shortcode,The%20name%20of%20the%20shortcode%2C%20provided%20for%20context%20to%20enable%20filtering,-Default%3A) `$shortcode` for this. Should we also follow that standard?

Reply:
I also don't see that parameter being used.

Reply:
Core uses `$tag` elsewhere, so I don't think it's really a standard. This parameter is only useful if we're using the same callback for different shortcodes, which doesn't apply here. I'll remove it.